### PR TITLE
EOS-20455: Stabilize sspl-test in CI-CD

### PR DESCRIPTION
**Pull request by mariyappanp**

### CODE REVIEW

File: sspl_test/alerts/node/test_node_bmc_interface.py [hunk 1]
```diff
@@ -43,21 +43,25 @@
         simulate_bmc_interface_alert.kcs_channel_alert(BMC_IF_CONSUL_KEY,BMC_IF_CONSUL_VAL)
 
     bmc_interface_message = None
-    time.sleep(25)
-    while not world.sspl_modules[IngressProcessorTests.name()]._is_my_msgQ_empty():
-        ingressMsg = world.sspl_modules[IngressProcessorTests.name()]._read_my_msgQ()
-        time.sleep(0.1)
```
Comment (thavanathan) [inline sspl_test/alerts/node/test_node_bmc_interface.py:49]: We still need this line inside while loop.

File: sspl_test/alerts/node/test_node_disk_actuator.py [hunk 2]
```diff
@@ -31,25 +31,26 @@ def init(args):
...
+    disk_actuator_message_request("NDHW:node:fru:disk", instance_id)
     disk_actuator_msg = None
-    time.sleep(10)
     ingressMsg = {}
     for i in range(10):
         if world.sspl_modules[IngressProcessorTests.name()]._is_my_msgQ_empty():
             time.sleep(2)
         while not world.sspl_modules[IngressProcessorTests.name()]._is_my_msgQ_empty():
             ingressMsg = world.sspl_modules[IngressProcessorTests.name()]._read_my_msgQ()
-            time.sleep(0.1)
```
Comment (thavanathan) [inline sspl_test/alerts/node/test_node_disk_actuator.py:43]: we still need this inside while loop.

File: sspl_test/alerts/node/test_node_psu_actuator.py [hunk 3]
```diff
@@ -31,23 +31,29 @@
...
     check_sspl_ll_is_running()
-    psu_actuator_message_request("NDHW:node:fru:psu", "*")
+    instance_id = "*"
+    psu_actuator_message_request("NDHW:node:fru:psu", instance_id)
     psu_module_actuator_msg = None
-    time.sleep(6)
     ingressMsg = {}
-    while not world.sspl_modules[IngressProcessorTests.name()]._is_my_msgQ_empty():
-        ingressMsg = world.sspl_modules[IngressProcessorTests.name()]._read_my_msgQ()
-        time.sleep(0.1)
```
Comment (thavanathan) [inline sspl_test/alerts/node/test_node_psu_actuator.py:40]: we still need this inside while loop.

File: sspl_test/alerts/os/test_systemd_service_actuator.py [hunk 4]
```diff
@@ -34,23 +34,28 @@
     # TODO: Change service name, once get final 3rd party service name
     service_actuator_request(service_name, request)
     service_actuator_msg = None
-    time.sleep(8)
     ingressMsg = {}
-    while not world.sspl_modules[IngressProcessorTests.name()]\
-                                                        ._is_my_msgQ_empty():
-        ingressMsg = world.sspl_modules[IngressProcessorTests.name()]\
-                                                    ._read_my_msgQ()
-        time.sleep(0.1)
```
Comment (thavanathan) [inline sspl_test/alerts/os/test_systemd_service_actuator.py:43]: move this inside while loop.

File: sspl_test/alerts/node/test_node_bmc_interface.py [hunk 5]
```diff
@@ -43,21 +43,25 @@ def test_bmc_interface(args):
...
+        if world.sspl_modules[IngressProcessorTests.name()]._is_my_msgQ_empty():
+            time.sleep(2)
+        while not world.sspl_modules[IngressProcessorTests.name()]._is_my_msgQ_empty():
+            ingressMsg = world.sspl_modules[IngressProcessorTests.name()]._read_my_msgQ()
+            print("Received: %s" % ingressMsg)
+            try:
+                # Make sure we get back the message type that matches the request
+                msg_type = ingressMsg.get("sensor_response_type")
+                if msg_type["info"]["resource_type"] == "node:bmc:interface:kcs" or \
+                    msg_type["info"]["resource_type"] == "node:bmc:interface:rmcp":
```
Comment (jjxsg) [inline sspl_test/alerts/node/test_node_bmc_interface.py:56]: `if msg_type["info"]["resource_type"] in ["node:bmc:interface:kcs","node:bmc:interface:rmcp"]:` can be used for or conditions.

File: sspl_test/alerts/node/test_node_bmc_interface.py (hunk 1 above)
Comment (mariyappanp) [reply sspl_test/alerts/node/test_node_bmc_interface.py:49]: I didn't remove any lines except time.sleep.

File: sspl_test/alerts/node/test_node_disk_actuator.py (hunk 2 above)
Comment (mariyappanp) [reply sspl_test/alerts/node/test_node_disk_actuator.py:43]: Why?

**Review by shriya-deshmukh — APPROVED** [review]

**Review by cdeshmukh — APPROVED** [review]

**Pull request merged**